app/views/newPaymentFlow.scala.html

```diff
@@ -58,7 +58,10 @@ <h1>Help us deliver the independent journalism the world needs</h1>
 
   <div>
     <label for="contributionLName">Last Name</label>
-    <input id="contributionLName" type="text" required>
+    <input id="contributionLName" type="text" aria-describedby="err1" required>
+    <div id="err1">
```

Comment:
Could we use the full `contributionLastName`, and give the error div a more descriptive `id`?

Reply:
Sure thing! I didn't know whether you had some naming logic in React, so I made them up, but if they stay like that then they need better names indeed.